soc/xtensa/intel_adsp/common/include/cpu_init.h

```diff
@@ -113,7 +113,7 @@ static ALWAYS_INLINE void cpu_early_init(void)
 	 * local CPU!  We need external transactions on the shared
 	 * bus.
 	 */
-	reg = CONFIG_MP_NUM_CPUS == 1 ? 0 : 0x15;
+	reg = 0x15;
```

Comment:
I'm sure this is fine, but it stood out to me in reading this. Can you comment on why this is being removed?

Reply:
c.f. huge essay in the commit message :)

Reply:
Of course! I feel like github review doesn't always point me at the commit messages and log, I should look at those more on review. Will look, thanks!